arboretum/permissions/README.md

```diff
@@ -94,6 +94,53 @@ how to include the fetchers and checks from this library in your downstream proj
 
 ## Checks
 
+### Organization Integrity (Repository Permissions)
+
+* Class: [OrgPermissionsCheck][org-permissions-check]
+* Purpose: Ensure that `direct` collaborators and forks do not exist in the organization repositories.
+* Behavior: Collaborators and forks are checked for every repository. A failure is generated when `direct` collaborators are found in a repository. A warning is generated for each fork found for a repository. 
```

Comment:
It looks like it's direct and outside collaborators that generate the failure.  Am I missing something?

Reply:
Failure is genereted for direct collaborators, outside collaborators are used to determine if a direct is org member or not.